src/tools/perf/libperf.c

```diff
@@ -928,7 +944,8 @@ static void ucp_perf_test_free_contig(ucx_perf_context_t *perf, void *addr, ucp_
         if (status != UCS_OK) {
             ucs_warn("ucp_mem_unmap() failed: %s", ucs_status_string(status));
         }
-    } else if (perf->params.mem_type == UCT_MD_MEM_TYPE_CUDA) {
+    } else if (perf->params.mem_type == UCT_MD_MEM_TYPE_CUDA
```

Comment:
Please guard each statement with `()` and move `||` to the first line

Reply:
@yosefe this doesn't apply. There's just one condition

Reply:
@yosefe this doesn't apply. There is just one condition.